tests/doc_get.rs

```diff
@@ -1,15 +1,17 @@
-use crate::features::TestFeature;
-use crate::util::playground;
-use crate::{cbsh, ClusterUnderTest, ConfigAware};
+mod common;
+
+use crate::common::{features::TestFeature, playground::CBPlayground, utils, ConfigAware};
 use nu_test_support::pipeline;
-use std::sync::Arc;
 
-pub async fn test_get_a_document(cluster: Arc<ClusterUnderTest>) -> bool {
+#[test]
+fn test_get_a_document() {
+    let cluster = utils::test_cluster();
+
     if !cluster.config().supports_feature(TestFeature::KeyValue) {
-        return true;
+        return;
     }
 
-    playground::CBPlayground::setup("get_a_document", cluster.config(), None, |dirs, sandbox| {
+    CBPlayground::setup("get_a_document", cluster.config(), None, |dirs, sandbox| {
```

Comment:
Given we're going to use a global variable effectively now it'd be cool if CBPlayground::setup also just returned the cluster to us rather than needing us to supply it.  We can just tell it which feature(s) we're testing as a part of the parameters we pass in. The tricky part is how to signal from setup that the test should be skipped.

Reply:
Done an initial iteration of this. One issue is that we call setup when starting the standalone cluster when we wait for kv/scope/collection. This is why I have added an Option to pass in the config, since in those cases we need to pass the parsed config in, otherwise you get stuck in a infinite loop, but there is probably a cleaner way to handle it. 

If a test feature is not supported then setup just returns, which will mark the tests as passed, not ignored. This is the same behaviour as before the change, so no worse off. Unfortunately I couldn't find a way to ignore tests at runtime, but I'll take another stab at it tomorrow.